base/exports.jl

```diff
@@ -1214,6 +1214,7 @@ export
     @__FILE__,
     @__DIR__,
     @__LINE__,
+    propagate_sourceloc!,
```

Comment:
Good question.  That depends on whether there's a distinction between "public API" and "exported function". I want this as a tool which can be used by packages, but it's also relatively obscure. So if there's a well defined category "public but not exported" this would fit in there, and we could not bother to export it.